### ISSUE TITLE
Instructions for building under MSYS2

### ISSUE THREAD

**Issue by wbhart** [wbhart/mpir#155]
The following will be added to the mpir.org website. It is a set of instructions for building MPIR under MSYS2. Both 32 and 64 bit versions have been tested.

Note: some virus scanners will conflict with msys2. I had to
temporarily disable mine, otherwise it gave permission denied
errors.
- Follow instructions here:
  
  https://msys2.github.io/

Start the msys2 shell again, from the Start Menu
- pacman -S wget gcc vim tar make texinfo git diffutils
- git clone https://github.com/wbhart/mpir mpir-2.7.0
- cd mpir-2.7.0
- ./configure --enable-gmpcompat
- make -j
- make -j check

**Comment by ostroffjh** [comment]
In trying to build mpir for KDE for Windows using their craft (previously named emerge) build system on Windows 7 32 bit.  The configure step fails with "error: Oops, mp_limb_t doesn't seem to work" apparently because it calls configure using an absolute path, not relative.  (I reported this in August to the KDE for Windows list:  https://mail.kde.org/pipermail/kde-windows/2016-August/009844.html)  I know I can manually do ./configure, but it would be far easier if it weren't necessary.  I have also seen this problem reported for gmp, but the patch I found for it (https://gmplib.org/list-archives/gmp-bugs/2013-July/003100.html) does not apply to mpir (2.7.2).  I might be able to sufficiently massage that patch, but my understanding is sufficiently limited, and I'm afraid I'll mess up something else.  Other than installing manually, is there any known solution to this problem?
